### DISCUSSION TITLE
SONOFF Smart Power Meter Switch | POW Elite Wifi LED

### DISCUSSION THREAD

**Discussion by lakusz1github** [arendst/Tasmota#19188] [Q&A] [answered]
Hi

After flash tasmota 13.0.0. to POWR316D witch success, and paste configuration template, every things works fine, except Wifi led not light blue when device is connected to AP. Some one know how to fix it ?

**Comment by lakusz1github** [comment]
After power it on, led ones blinked and its "not report" the wifi connection is ok or not

**Comment by sfromis** [comment, marked as answer]
Default Tasmota behavior is to blink if lacking a network connection, and off when ok. IMO this is more sensible than having static leds go off. You do have a number of options to change the behavior. https://tasmota.github.io/docs/Lights/#status-leds
